project/generate-authors.sh

```diff
@@ -11,5 +11,5 @@ cd "$(dirname "$(readlink -f "$BASH_SOURCE")")/.."
 	# For how it is generated, see `project/generate-authors.sh`.
 	EOH
 	echo
-	git log --format='%aN <%aE>' | sort -uf
+	git log --format='%aN <%aE>' | LC_ALL=C sort -uf
```

Comment:
I ran the script again with `C.UTF-8` which seems to work fine. The names were moved around due to the locale change

```bash
$ LC_ALL=en_US sort <<< $'a\nb\nA\nB'
a
A
b
B
$ LC_ALL=C sort <<< $'a\nb\nA\nB'
A
B
a
b
```